.gitignore

```diff
@@ -70,3 +70,4 @@ atlassian*
 /.docker/elasticsearch
 /.docker/.env
 /.docker/php/conf.d/custom.ini
+composer.phar
```

Comment:
We don't use this file in the project so it's not needed to add it to `.gitignore`.